### PR TITLE
Feature/65 hide modules in navigation

### LINKED ISSUE #55: Help Does Not Appear When No Modules
We do not load the help widget if modules is `[]`. This is inconsistent, because we load the help widget with the default product (General) if modules are not passed in at all. We should do the same if an empty array is passed in. 

This is particularly bad for users with only user management rights, as we are no longer counting Users as a product, so they technically have no products. But, they should still see the help widget with general support.

When fixing this, we should be careful to avoid breaking the async loading of modules. We should make sure consumers pass in something other than `[]` while they load modules, or we will load the help erroneously with just the default product.

### LINKED ISSUE #65: Hide Users Product in Navigation
Currently, the Solution Center is hiding the Users product by excluding it from the modules passed into the header. This is not ideal, since all products would need to add this exclusion logic in their frontend. The preferred solution is for each module to have a `hidden` property, and the header will hide them accordingly. This has several benefits including being a centralized solutions (just a simple filter in the header) and it allows products to be toggled via API call rather than requiring a redeploy.

Once the `hidden` property is implemented on the backend, hide these modules from the navigation.

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #65 and fixes #55

### CODE REVIEW

**Comment by dami-gg** [comment]
We'll release a new version as soon as the backend guys do their part

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "verify": "npm run verify:npm",
     "verify:npm": "npm view $npm_package_name version",
     "verify:bower": "bower info $npm_package_name version",
-    "dev": "npm run build && concurrently \"npm run serve\" \"npm run watch\" "
+    "dev": "npm run build && concurrently \"npm run start\" \"npm run watch\" "
```
Comment (cjhowald) [inline package.json]: Personal preference, but I would leave the `serve` task and change `dev` to `start`.

**Comment by cjhowald** [comment]
Test pass - debug toggle works as expected.

**Comment by cjhowald** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**